bip-ctv.mediawiki

```diff
@@ -244,27 +244,27 @@ Below we'll discuss the rules one-by-one:
 The set of data committed to is a superset of data which can impact the TXID of the transaction,
 other than the inputs. This ensures that for a given known input, the TXIDs can also be known ahead
 of time.  Otherwise, CHECKTEMPLATEVERIFY would not be usable for Channel Factory type constructions
-as the redemption TXID could be malleated and pre-signed transactions invalidated.
```

Comment:
I think malleated might have a specific meaning here (that probably should be elaborated on for clarity). So changing that word to "change" might not be the right thing here.

Reply:
I assume that it is trying to refer to »transaction malleability« but it's the first time I've ever seen »malleated« and it just feels like change conveys the same meaning but is clearer.

Reply:
I assume so as well. I have seen other people use the word malleated as well in cases like this. You can see its commonality with a [google search](https://www.google.com/search?q=bitcoin+malleated&rlz=1C1CHBF_enUS874US874&oq=bitcoin+malleated&aqs=chrome..69i57j0l7.1623j0j7&sourceid=chrome&ie=UTF-8).